src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
+
+use crate::prelude::*;
+
```

Comment:
The first breadcrumb cannot be selected:
    ![2020-08-06 10 23 19](https://user-images.githubusercontent.com/1623053/89509031-0e84ae80-d7cf-11ea-913b-daf2a1422ca5.gif)